RockPaperScissors/RockPaperScissors/RockScissorPaper.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  RockPaperScissors - main.swift
+//  Created by Lingo, 우롱차.
+//  Copyright © yagom academy. All rights reserved.
+//
+
+import Foundation
+
+enum HandType: Int, CaseIterable {
+    case scissor = 1
+    case rock = 2
+    case paper = 3
+    
+    static var randomPick: Int? {
+        return Self.allCases.randomElement()?.rawValue
+    }
+    
+    static func isHandType(_ value: Int) -> Bool {
+        if let _ = HandType(rawValue: value) {
+            return true
+        }
+        return false
+    }
+    
+    static func convertMukZziPpaToHandType(_ value: Int) -> Int? {
+        switch value {
```

Comment:
묵찌빠 손타입을 가위바위보 손타입으로 변환해주는 타입 메서드를 만드셨군요! 👍🏻 

지난 코멘트에서도 말씀드렸지만, `HandType`의 타입 메서드이므로
호출될 때 `HandType.convertMukZziPpaToHandType` 와 같이 불리게 될 겁니다.
"HandType" 이라는 단어가 중복되죠? ㅎㅎㅎ

차라리 손모양을 `묵찌빠 스타일`로 전환하다(convert) 라는 의미를 주는 건 어떨까요? 😄 

```suggestion
    static func convertMukZziPpaStyle(_ value: Int) -> Int? {
        switch value {
```

Reply:
처음 네이밍을 할때 **묵찌빠 형식을 HandType 열거형의 case에 맞게 변환해준다**는 의미로 `convertMukZziPpaToHandType` 라고 했었는데 피드백 해주신 `~스타일` 라는 네임으로 리팩토링 했습니다.
역시 네이밍이 가장 어렵네요 😅